docs/source/tutorials.rst

```diff
@@ -78,6 +68,29 @@ Gaussian filtering and smoothing in more complex dynamical systems.
    tutorials/filtsmooth/particle_filtering
 
 
+Linear Operators
+----------------
+
+Linear algebra is fundamental to arguably most of scientific computation. ProbNum defines (finite-dimensional) linear
```

Comment:
```suggestion
Linear algebra is fundamental essentially all of scientific computation. ProbNum implements (finite-dimensional) linear operators in a memory-efficient manner with support for lazy arithmetic.
```

Reply:
That first part of the sentence has a word missing I believe, and "fundamental" and "essentially" sound a bit redundant to me. What about 

"Linear algebra is fundamental to virtually all of scientific computation." 

or 

"Linear algebra is essentially to virtually all of scientific computation."

I'll change the second sentence as proposed.

Reply:
I used the second version.